doc/documentation.rst

```diff
@@ -28,8 +28,9 @@ Documentation of scikit-learn 0.18.dev0
                 <!-- doc versions -->
                     <h2>Other Versions</h2>
                     <ul>
-                        <li>scikit-learn 0.18 (development)</li>
-                        <li><a href="http://scikit-learn.org/stable/documentation.html">scikit-learn 0.17 (stable)</a></li>
+                        <li>scikit-learn 0.18 (stable)</li>
+                        <li><a href="http://scikit-learn.org/dev/documentation.html">scikit-learn 0.19 (development)</a></li>
+                        <li><a href="http://scikit-learn.org/0.17/documentation.html">scikit-learn 0.17</a></li>
```

Comment:
ok, fair. I'm actually not sure how we currently built the stable website. Do you know?